fronts-client/src/components/feed/RecipeSearchContainer.tsx

```diff
@@ -109,22 +134,33 @@ export const RecipeSearchContainer = ({ rightHandContainer }: Props) => {
         </TextInputContainer>
         <ClipboardHeader />
       </InputContainer>
-      <RadioGroup>
-        <RadioButton
-          checked={selectedOption === FeedType.recipes}
-          onChange={() => setSelectedOption(FeedType.recipes)}
-          label="Recipes"
-          inline
-          name="recipeFeed"
-        />
-        <RadioButton
-          checked={selectedOption === FeedType.chefs}
-          onChange={() => setSelectedOption(FeedType.chefs)}
-          label="Chefs"
-          inline
-          name="chefFeed"
-        />
-      </RadioGroup>
+      <TopOptions>
+        <RadioGroup>
+          <RadioButton
+            checked={selectedOption === FeedType.recipes}
+            onChange={() => setSelectedOption(FeedType.recipes)}
+            label="Recipes"
+            inline
+            name="recipeFeed"
+          />
+          <RadioButton
+            checked={selectedOption === FeedType.chefs}
+            onChange={() => setSelectedOption(FeedType.chefs)}
+            label="Chefs"
+            inline
+            name="chefFeed"
+          />
+        </RadioGroup>
+        {selectedOption === FeedType.chefs && chefsPagination && hasPages && (
+          <PaginationContainer>
+            <Pagination
+              pageChange={(page) => setPage(page)}
```

Comment:
At the moment, setPage triggers the effect that runs our search – but the result of that is that selecting pages feels quite slow, as we have to wait 750ms for the fetch to begin! Perhaps we could trigger a fetch immediately here, after updating the page number in the state?